data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+		conversation
+			`When you return to Hai-home you are met by a group of relieved diplomats.`
+			`	Of all of them Alondo seems to be the least tired and takes the time to let you know what's been going on, "Captain <last>! You've missed quite the palava. Still, I doubt you lost much by missing out on trying to explain to Sayari what a 'furry' is."`
+			choice
+				`	"Oh I don't know, could've been entertaining."`
+				`	"Hmm, yes, no probably not."`
+			`	He chuckles, "Anyway, I have a hunch that we're going to be needing you before too long, but for now they're putting some Hai patrols out on the other side of the wormhole with a couple of Navy observers on board to 'play the part', as it were, of an exclusive role-play community, just chilling out here and having fun."`
+			`	"Come though," he says, "I hear you've been instrumental in driving off some opportunistic pirates and helping stem the information leak, I'll buy you a drink on the other's behalf."`
+			`	Some hours later you leave the fancy cocktail bar Alondo took you to and begin making your way back to your ship to sleep, your cabin by far seems to be the best place to not be bothered.`
+			`	Part way there though a hooded figure suddenly falls in step beside you.`
+			choice
+				`	(Keep walking.)`
+				`	(Ask them who they are.)`
+			`	"Sssshhhh!" says the voice at your elbow, "do not react, and do not be alarmed, it is merely I."`
+			`	You look at the figure out of the corner of your eye and connect the whispered voice to the squirrelly muzzle poking out. To your surprise it is Sayari.`
+			`	"I should not be coming to you with this, so you must promise to say nothing of this to anyone just yet."`
+			choice
+				`	"Of course."`
+			`	"A very sweet friend of mine, who is much too enthusiastic and perhaps led by the vagaries of youth but nonetheless quite astute, keeps harping on to me about this theory of hers that a smuggling ring is kidnapping Hai and transporting them into human space."`
+			`	She pauses a moment for you to absorb this, "Of course, a thorough investigation has occurred on our side and no evidence of such things has turned up, but..."`
+			`	"Well, you see we cannot investigate your space openly or easily. I have paid it little mind, written it off as one of her eccentricities, but things are very tense right now. The Council of Elders wants to maintain our secrecy, the human diplomats are willing to accomodate us, but we sense their governments grow tired of maintaining the fiction. We are well aware that human intelligence services are highly competent, and motivated by many different factors. It worries me that one of your factions may not be wholly honest, and may have knowledge they intend to use as ammunition if things don't work out well for them."`
```

Comment:
This explanation seems a little sidetracked. Sayari doesn't really provide a clear explanation as to why she's bringing up this issue of her eccentric friend _now_, in the middle of the negotiations.

Reply:
If they're going to 'come out' to the world they want to know they're dealing honestly.

Reply:
Could you elaborate on what you mean? I don't really understand what you're saying.

Reply:
Say you're a diplomat, you represent a group that is essentially 'secret', and now it seems that you have to navigate through a process of representing them as they either unveil that secret or go to great lengths to preserve it.
There is no official information suggesting that there should be a problem with the people you're negotiating with, but you have a friend/contact who is respected and influential in their own circles that insists there is a problem. 

You are subsequently presented with the opportunity to deal with a single individual who appears apparently trustworthy and who has the means to look into it. Knowing that if it is indeed true it will impact what you're trying to do: Do you keep quiet and ignore this doubt that your friend keeps bringing up, or do you see if you can get this individual to help lay that concern to rest for good before it's too late?

Reply:
Oh, now I understand. Thanks for the clarification. I would say some of this explanation should get worked into this conversation, but that's just personal preference. Your choice.

Reply:
I could, and I may. I do like to leave some things for the canny reader to spot for themselves though. ;)